doc/dev/GETTING_STARTED.md

```diff
@@ -35,7 +35,7 @@ Note: Ubuntu 24.04 / Debian 12 provides VTK version 9.1 but f3d requires VTK 9.2
 
 ```
 sudo yum update
-sudo yum install make automake gcc gcc-c++ kernel-devel git git-lfs cmake vtk
+sudo dnf install make automake gcc gcc-c++ kernel-devel git git-lfs cmake vtk vtk-devel
```

Comment:
@upadhyayaniket29 should only `vtk-devel` be listed  ? it depends on vtk anyway.

```suggestion
sudo dnf install make automake gcc gcc-c++ kernel-devel git git-lfs cmake vtk-devel
```